ironfish-cli/src/commands/wallet/multisig/create-signing-commitment.ts

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { RpcAccountImport } from '@ironfish/sdk'
+import { Flags } from '@oclif/core'
+import { IronfishCommand } from '../../../command'
+import { RemoteFlags } from '../../../flags'
+
+export class CreateSigningCommitmentCommand extends IronfishCommand {
+  static description = `Attempt to connect to a peer through websockets`
+
+  static flags = {
+    ...RemoteFlags,
+    account: Flags.string({
+      char: 'a',
+      description: 'The account to use for the transaction',
+      required: false,
+    }),
+    transaction: Flags.string({
+      char: 't',
+      description: 'The unsigned transaction',
+      required: false,
+    }),
+  }
+
+  async start(): Promise<void> {
+    const { flags } = await this.parse(CreateSigningCommitmentCommand)
+
+    const client = await this.sdk.connectRpc()
+
+    const accountResponse = await client.wallet.exportAccount(
+      flags.account ? { account: flags.account } : {},
+    )
+    const account = accountResponse.content.account as RpcAccountImport
+
+    if (!account.multiSigKeys) {
+      this.error(`Account "${account.name}" is not a multisig account`)
+    }
+
+    // TODO(andrea): use flags.transaction to create commiment when we incorportate deterministic nonces
+    // set required to true as well
+    const commitmentResponse = await client.multisig.createSigningCommitment({
+      keyPackage: account.multiSigKeys.keyPackage,
+      seed: 0,
+    })
+
+    this.log('Commitment:\n')
+    this.log(JSON.stringify(commitmentResponse.content))
```

Comment:
Do we want to serialize as JSON, or base 64, or what?

Reply:
I think JSON might be the easiest to work with for now 🤔 

The input to the next command will need to be a list of identifiers paired with commitments. If the coordinator needs to assemble that list it might be easiest to collect everything in a JSON file